makefiles/features_precheck.inc.mk

```diff
@@ -0,0 +1,17 @@
+# Check if provided features are available
+
+ifneq (,$(filter picolibc,$(FEATURES_PROVIDED)) $(filter newlib,$(FEATURES_PROVIDED)))
```

Comment:
The outer if can be dropped, can't it?

Reply:
No it can (i wanted to avoid the rest of the check if not nesecary but the shell checks a only done if the feature is provided).

I will remove it